app/models/events/comment_announced.rb

```diff
@@ -0,0 +1,29 @@
+class Events::CommentAnnounced < Event
+  include Events::Notify::InApp
+  include Events::Notify::ByEmail
+
+  def self.publish!(model, actor, audiences)
+    super model,
+          user: actor,
+          custom_fields: { audiences: }
+  end
+
+  private
+
+  def email_recipients
+    notification_recipients.where(email_when_mentioned: true)
+  end
+
+  def notification_recipients
+    recipients_ids = custom_fields['audiences'].inject([]) do |ids, audience|
+                      ids |= AnnouncementService.audience_users(eventable,
+                                                                Audience.send(audience).alias,
+                                                                eventable.author,
+                                                                false,
```

Comment:
you can remove false, false from this. They're defaults, sorry that they're not named arguments, they really should have been for readability, but seeing as they're not used, just remove false, false.